docs/application/native/guides/internationalization/i18n.md

```diff
@@ -809,12 +809,12 @@ Plural Format deals with this by breaking the problem into two parts:
 
 ### Usage of Plural Format:
 
-This discussion assumes that you use Plural Format with a predefined set of plural rules.
-You can create one using one of the constructors that takes a locale object.
-To specify the message pattern, you can either pass it to the constructor or set it explicitly
-using the `i18n_plural_format_apply_pattern()` function.
-The `i18n_plural_format_format_*` functions takes a number object and selects the message
-of the matching plural case. This message will be returned.
+To properly use Plural Format module, you will need to use Plural Rules module, to obtain
+predefined sets of rules. To get default set of rules, use constructor which takes locale object
+as parameter. Specifying message pattern is done by passing argument to constructor or to set it
+explicitly by using the `i18n_plural_format_apply_pattern()` function.
+To obtain formatted message, you need to use one of the `i18n_plural_format_format_*` functions,
+which takes also a number object and returns corresponding message based on the defined rules.
 
```

Comment:
Please change as following:

The following lists the usage of Plural format:
1.	You will need to use Plural module to properly use PluralFormat module.
2.	To get default set of rules, you can use constructor, which takes local object as parameter. 
3.	A message pattern is specified by passing argument to constructor, or set it explicitly by using 18n_plural_format_apply_pattern().
4.	To obtain formatted message, you need to use one of i18n_plural_format_format_*, which also takes a number object and returns corresponding message based on the defined rules.